projects/components/src/tabs/content/tab/tab.component.ts

```diff
@@ -13,4 +13,7 @@ export class TabComponent extends ContentHolder {
 
   @Input()
   public labelTag?: string;
+
+  @Input()
+  public info?: string | TemplateRef<unknown>;
```

Comment:
Similar to how we have **context** for **Tooltip**, would it make sense to have similar logic implemented?

Reply:
I am not exactly sure about this, I am hoping by this, parents would take care of that